types/types.go

```diff
@@ -545,6 +545,9 @@ type SystemContext struct {
 	CompressionFormat *compression.Algorithm
 	// CompressionLevel specifies what compression level is used
 	CompressionLevel *int
+
+	// Allows overwrite default temporary directory for storing big files,
+	BigFilesTemporaryDir string
```

Comment:
- This should be in the “global configuration overrides” section.
- The comment should be `If not "", overrides the temporary directory to use for storing big files` or something like that, to be consistent with the others and to clearly how to not override the default.